cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -45,6 +45,7 @@
 #include <utility>
 
 namespace cudf {
+
 namespace experimental {
```

Comment:
```suggestion
namespace cudf::experimental {
```

Reply:
This file has several layers of nested namespaces. For consistency, I would recommend leaving this unchanged.